jquery.xblock.js

```diff
@@ -59,7 +59,10 @@
             lmsSecureURL: false, // Is the LMS on HTTPS?
             useCurrentHost: false, // set to true to load xblock using the current location.hostnam
             disableGlobalOptions: false, // set to true to disable the global_options behavior.
-            data: {}              // additional data to send to student_view. send as GET parameters
+            data: {},              // additional data to send to student_view. send as GET parameters
+            rewriter: function (jumpToLink) {} // Function to rewrite jump links if needed for your target platform.
```

Comment:
@Kelketek `rewriter` sounds very generic, how would you feel about changing the option name to something like `jumpLinkRewriter`?